Infrastructure/Configuration/FeatureFlags.cs

```diff
@@ -6,5 +6,8 @@ public class FeatureFlags
 
     public bool VerifyEmail { get; set; } = true;
     public bool EnquiryBuilder { get; set; } = true;
+    public bool SendEmailsFromNtp { get; set; } = true;
+    public bool VerifyEmail { get; set; } = true;
```

Comment:
We don't need to add `VerifyEmail` as we already have it.

Reply:
Good spot - sorted